compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/align-scopes-nested-block-structure.ts

```diff
@@ -41,7 +38,7 @@
  *  │return s;  │◄──┘
  *  └───────────┘
```

Comment:
important question: how did you make this diagram?

Reply:
I used https://asciiflow.com with a little manual editing!